packages/components/filters/src/filter-menu/chip/chip.tsx

```diff
@@ -1,5 +1,40 @@
-function Chip() {
-  return <div>chip</div>;
+import { type ReactNode } from 'react';
+import { designTokens } from '@commercetools-uikit/design-system';
+import { css } from '@emotion/react';
+
+export type TFilterMenuChipProps = {
+  isDisabled?: boolean;
+  label: ReactNode;
+};
+
+const chipStyles = css`
+  font-size: ${designTokens.fontSize20};
+  font-weight: ${designTokens.fontWeight400};
+  line-height: ${designTokens.lineHeight20};
+  color: ${designTokens.colorPrimary20};
+  background-color: ${designTokens.colorPrimary95};
+  height: ${designTokens.spacing40};
+  padding: 0 ${designTokens.spacing20};
+  border-radius: calc(
+    ${designTokens.borderRadius20} - ${designTokens.borderRadius4}
+  );
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  list-style: none;
+`;
+
+const disabledChipStyles = css`
+  color: ${designTokens.colorNeutral40};
+  background-color: ${designTokens.colorNeutral90};
+`;
+
+function Chip(props: TFilterMenuChipProps) {
+  return (
+    <li css={[chipStyles, props.isDisabled && disabledChipStyles]}>
```

Comment:
Personally, to keep this component portable, I'd go with a `span` tag  and wrap the `Badge` with an `li`-tag  when it's put to use. This way you can also make tweaks on the list separate from the `Badge` implementation (e.g. :first-child & :last-child adjustments)

Reply:
this component is specifically not meant to be portable, and only for use in this specific place, which is why I went with the `li`. Partly to reinforce that it is meant to be used specifically in a list.

If we need to make it its' own component at some point, we can refactor it then